src/libs/Network/SequentialQueue.ts

```diff
@@ -71,7 +70,7 @@ function process() {
         .then((response) => {
```

Comment:
Can't we make this have a type?

Reply:
The type is inferred from `Request.processWithMiddleware` function